src/oop/oop1.py

```diff
@@ -1,19 +1,49 @@
 # Write classes for the following class hierarchy:
```

Comment:
This looks good! Great work, @MelvinPeepers ! 🙌 
<img width="627" alt="melvin_oop1" src="https://user-images.githubusercontent.com/16110931/76147876-ccfca580-606e-11ea-8ceb-b2d8da33ffb6.png">